coursedata/adventures/en.yaml

```diff
@@ -2154,66 +2169,130 @@ adventures:
                     You have reached the end of this level, you're doing great! You are about to go to the next level. In the higher levels, Hedy is focussing more and more on teaching you the programming language Python.
                     In Python there is no `repeat` command, but there is a command that works like repeat. Are you curious to find out how to say `repeat` in Python language? Quickly go on to find out!
                 start_code: "print 'Lets go to the next level!'"
-# These are still empty, I'll fill them up soon.
             11:
                 story_text: |
                     ## What's next?
+                    Maybe you have tried using decimal numbers in your restaurant adventure. If you did, you probably noticed that Hedy didn't understand them yet and always rounded off.
+                    From the next level on you can use decimal numbers.
 
+                    Another cool feauture in the next level is that you can use mulitiple words in a variable.
+                    For intance, you could already do this...
+                    ```
+                    name = ask 'Who is your favorite cartoon character?'
+                    print 'I love watching ' name
+                    ```
+                    But you couldn't use the `if` command, like the code below. This program does not work yet in this level!
```

Comment:
Ha @MarleenGilsing!

Dit is niet meer waar want je mag nu in alle levels ook spaties in de if gebruiken, dat hebben we gefikst. Dus deze tekst moet een beetje anders.